include/boost/geometry/algorithms/detail/envelope/implementation.hpp

```diff
@@ -43,27 +46,31 @@ namespace detail { namespace envelope
 {
 
 
-struct exterior_ring_expand_policy
-{
-    template <typename Box, typename Geometry>
-    static inline void apply(Box& mbr, Geometry const& geometry)
-    {
-        Box ring_mbr;
-        envelope_range<>::apply(exterior_ring(geometry), ring_mbr);
-        geometry::expand(mbr, ring_mbr);
-    }
-};
-
-
+template <std::size_t Dimension, std::size_t DimensionCount>
```

Comment:
I have not yet implemented support for polygons in the spherical equatorial coordinate system, so it is hard to see where this goes. The basic idea is that for the spherical equatorial coordinate system, we need to compute the envelope of longitudes and latitudes, and then call the generic `envelope_polygon::apply()` method for the remaining coordinates.